test.js

```diff
@@ -7,7 +7,7 @@ const contentDir = path.join(__dirname, 'content')
 
 test('defines nodeVersions', () => {
   const majors = Object.keys(nodeVersions)
-  const expectedMajors = ['v12.x', 'v10.x', 'v8.x']
+  const expectedMajors = ['v13.x', 'v12.x', 'v10.x', 'v8.x']
```

Comment:
Drive-by comment 👋 

I think we originally discussed only doing translations for even-numbered versions of Node.js that will have LTS releases. Was that decision changed?

Reply:
Sorry, I've never seen/heard about that, so just added it. I think we should discuss this on weekly meeting, but now I'm thinking that's okay to translate each version.
I was made research about crowdin functionality, and now I'm pretty sure that translation process will be much easier than before. We'd just translate the difference between the versions (10-30 files usually), all other stuff will do automatically. So probably we'd like to change policy about even-numbered versions

Reply:
Sounds good to me.

> We'd just translate the difference between the versions (10-30 files usually), all other stuff will do automatically.

Nice. Are you thinking of using Crowdin's Translation Memory feature to fill in the gaps between versions?

Reply:
Yep, TM is a best way for that

Reply:
Should 8 be removed from here now that it is EOL?

Reply:
FYI  https://github.com/nodejs/i18n/pull/83  https://github.com/nodejs/i18n/pull/84